Makefile

```diff
@@ -187,6 +187,7 @@ minikube-conbench-url:
 start-minikube:
 	minikube delete --profile mk-conbench
 	minikube start --profile mk-conbench --cpus=2 --memory=4g \
+		--driver=docker \
```

Comment:
On my machine minikube suddenly started prompting me for a superuser password while picking the specific 'driver' it wanted to use. I got this prompt non-deterministically. I have the hope that being explicit about the driver helps.